Demo/Demo/Helpers/OrderRequestHelpers.swift

```diff
@@ -1,104 +1,97 @@
+                            countryCode: "US",
                             postalCode: "32422"
                         ),
+                        name: Shipping.Name(fullName: "Cookie Monster"),
                         options: shippingPreference == .getFromFile ? getShippingMethods() : nil
-                    )
+                    ),
+                    payee: Payee(merchantID: "X5XAHHCG636FA", emailAddress: "merchant@email.com"),
+                    amount: Amount(currencyCode: "USD", value: String(orderAmount), breakdown: nil)
                 )
-            ],
-            applicationContext: OrderApplicationContext(
-                shippingPreference: shippingPreference,
-                userAction: .payNow,
-                returnUrl: "https://example.com/return",
-                cancelUrl: "https://example.com/cancel"
-            )
+            ]
         )
     }
-    
-    static func getAmount(value: Double = orderAmount, shipping: Double = 3.99) -> PayPalCheckout.PurchaseUnit.Amount {
-        return PayPalCheckout.PurchaseUnit.Amount(
-            currencyCode: .usd,
-            value: String(value + shipping),
-            breakdown: PayPalCheckout.PurchaseUnit.Breakdown(
-                itemTotal: UnitAmount(currencyCode: .usd, value: String(value)),
-                shipping: UnitAmount(currencyCode: .usd, value: String(shipping))
+
+    static func getAmount(shipping: Double = 3.99) -> Amount {
+        Amount(
+            currencyCode: currencyCode,
+            value: String(orderAmount + shipping),
+            breakdown: Amount.Breakdown(
+                shipping: ItemTotal(value: String(shipping), currencyValue: String(shipping), currencyCode: currencyCode),
+                itemTotal: ItemTotal( value: String(orderAmount), currencyValue: String(orderAmount), currencyCode: currencyCode)
             )
         )
     }
-
-    static func getShippingMethods(baseValue: Int = 0, selectedID: String = "ShipTest1") -> [PayPalCheckout.ShippingMethod] {
-        let currency = CurrencyCode.usd
-        let ship1 = PayPalCheckout.ShippingMethod(
-            id: "ShipTest1",
-            label: "standard shipping",
-            selected: false,
-            type: .shipping,
-            amount: UnitAmount(currencyCode: currency, value: String(3.99 + Double(baseValue)))
-        )
-        let ship2 = PayPalCheckout.ShippingMethod(
-            id: "ShipTest2",
-            label: "cheap shipping",
+    
+    static func getShippingMethods(selectedID: String = "ShipTest1") -> [ShippingOption]? {
+        let shipOption1 = ShippingOption(
             selected: false,
-            type: .shipping,
-            amount: UnitAmount(currencyCode: currency, value: String(0.99 + Double(baseValue)))
+            id: "ShipTest1",
+            amount: ItemTotal(value: "3.99", currencyValue: "3.99", currencyCode: currencyCode),
+            label: "Standard Shipping",
+            type: "SHIPPING"
```

Comment:
Should there be an enum in our SDK for this? Not necessary for this PR but looks cleaner with `.shipping` in the checkout SDK

Reply:
Great question - so this helper function is formatting the POST body for a `PATCH` API call to orders v2 (see public docs https://developer.paypal.com/docs/api/orders/v2/#orders_patch). The merchant is responsible for doing this on their server after the shipping information changes in the MXO flow. Our sample server is just propagating whatever POST body we're sending on the client, through to orders/v2, but I imagine most of this logic would live on a merchant server instead.

That said, we could offer this enum, but to me feels out of the mobile SDK's domain since it is part of a server-side call they have to make.

What do you think?

Reply:
Another option too is an enum internal to the Demo app